tensorflow_addons/image/transform_ops.py

```diff
@@ -57,14 +61,18 @@ def transform(
       interpolation: Interpolation mode.
         Supported values: "NEAREST", "BILINEAR".
       fill_mode: Points outside the boundaries of the input are filled according
-        to the given mode (one of `{'constant', 'reflect', 'wrap'}`).
+        to the given mode (one of `{'constant', 'reflect', 'wrap', 'nearest'}`).
         - *reflect*: `(d c b a | a b c d | d c b a)`
           The input is extended by reflecting about the edge of the last pixel.
         - *constant*: `(k k k k | a b c d | k k k k)`
           The input is extended by filling all values beyond the edge with the
           same constant value k = 0.
         - *wrap*: `(a b c d | a b c d | a b c d)`
           The input is extended by wrapping around to the opposite edge.
+        - *nearest*: `(a a a a | a b c d | d d d d)`
```

Comment:
Does this actually work? According to the docs for `ProjectiveTransformV3` the `'nearest'` option is for the `interpolation` arg, not the `fill_value` arg. There is no test for `nearest` in `transform_ops_test.py` either.

Reply:
Yes it works, I forgot to add cc documentation in old PR :-(

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/image/image_ops.h#L113-L119